src/components/TestQuestion.js

```diff
@@ -9,9 +9,9 @@ function TestQuestion(props) {
            <div className={styles.question}>{question}</div>
            <div className={styles.answears}>
                <ul>
-                   <li>{answear1}</li>
-                   <li>{answear2}</li>
-                   <li>{answear3}</li>
```

Comment:
IMO you can add a .map method here to make this(heh this) component more reusable